server/api.go

```diff
@@ -66,7 +66,7 @@ func (p *Plugin) getGo(w http.ResponseWriter, r *http.Request) {
 	}
 
 	target := r.URL.Query().Get("target")
-	if len(target) != 2 {
+	if len(target) < 2 || len(target) > 5 {
```

Comment:
Are the code that have the length 3 or 4?

Reply:
Looking at some typical reference online and [supported language by AWS Translate](https://docs.aws.amazon.com/translate/latest/dg/what-is.html#language-pairs), it's either:
- 2 (ex. `fr`), or
- 5 (ex. `fr-CA`), including dash

Reply:
I don't think there are len 3 or 4 codes, only 2 and 5